edx_arch_experiments/kafka_consumer/docs/decisions/0001_kafka_based_event_bus.rst

```diff
@@ -0,0 +1,154 @@
+=====================
+Kafka-Based Event Bus
+=====================
+
+Status
+------
+
+Provisional
+
+Context
+-------
+
+The draft `OEP-52: Event Bus Architecture`_ explains how the Open edX platform would benefit from an event bus, as well as providing some additional decisions around the event bus. One decision is to enable the event bus technology to be pluggable through some abstraction layer.
+
+This still requires selecting a specific technology for the first implementation.
+
+.. _`OEP-52: Event Bus Architecture`: https://github.com/openedx/open-edx-proposals/pull/233
+
+Decision
+--------
+
+An initial implementation of an event bus for the Open edX platform will be implemented using `Kafka`_. This implementation will be used by edx.org, and available to the Open edX community.
+
+This decision does not preclude the introduction of alternative event bus implementations based on other technologies in the future.
+
+.. _Kafka: https://kafka.apache.org/
+
+Why Kafka?
+~~~~~~~~~~
+
+Kafka is a distributed streaming platform. Kafka's implementation maps nicely to the pub/sub pattern. However, some native features of a message broker are not built-in.
+
+Kafka has been around for a long time. See `Thoughtworks's technology radar introduced Kafka`_ as "Assess" in 2015, and "Trial" in 2016. It never moved up to "Adopt", and also never moved down to "Hold". Read `Thoughtwork's Kafka decoder page`_ to learn more about its benefits and trade-offs, and how it is used.
+
+More recently, the `Thoughtworks's technology radar introduced Apache Pulsar`_ as "assess" in 2020, and the `technology radar introduced Kafka API without Kafka`_ in 2021. This both demonstrates the de facto standard of the Kafka API, but also Thoughtwork's hope to find a less complex alternative.
```

Comment:
First sentence is missing a verb.

Reply:
Do you mean the sentence beginning with "More recently"? The verb is "introduced", and appears twice, although both times in the midst of a link, which may be why it was missed?